budgetrc.sample

```diff
@@ -0,0 +1,31 @@
+# Create a file ".budgetrc" in your home directory.
+# Then copy-paste this file (or whatever you need)
+# Lines with a '#' are comments and won't be read
+# Keep in mind, empty values like "default_currency=" will be valid (so in this case your currency is a currency without code/name :D)
+
+# VERY IMPORTANT --> Don't put comments on the right side of the '=' sign as they will be read as values.
+
+# Server url
+server_url=localhost
+# Port number
+server_port=8080
+# Currency code like EUR, USD...
+default_currency=CHF 
```

Comment:
Could you just move `default_account` out of this block so that server information are grouped together?